src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -58,7 +58,7 @@ class Dashboard extends AsyncComponent {
     const {teams, projects, params} = this.props;
     const sortedProjects = sortProjects(projects);
     const {projectsByTeam} = getProjectsByTeams(teams, sortedProjects);
-    const projectKeys = Object.keys(projectsByTeam);
+    const projectKeys = Object.keys(projectsByTeam).sort();
```

Comment:
shouldn't this variable be called "team keys"?

Reply:
Ah yeah, that'd probably make more sense